tests/usermetadata/test_metadata.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from pytest import mark
+from supertokens_python import InputAppInfo, SupertokensConfig, init
+from supertokens_python.querier import Querier
+from supertokens_python.recipe import usermetadata
+from supertokens_python.recipe.usermetadata.recipe import UserMetadataRecipe
+from supertokens_python.utils import compare_version
+from tests.utils import clean_st, reset, setup_st, start_st
+
```

Comment:
Thanks. Added a test for overrides. Others overlapped with `test_that_usermetadata_recipe_works_as_expected` so I didn't find adding them to be much useful. Let me know if you think otherwise.